src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -162,11 +166,10 @@ class JesBackend extends Backend with LazyLogging {
     val jesParameters = standardParameters(callGcsPath) ++ jesInputs ++ unsafeJesOutputs
 
     // Not possible to currently get stdout/stderr so redirect everything and hope the WDL isn't doing that too
-    val redirectedCommand = s"$commandLine > $LocalStdoutValue  2> $LocalStderrValue"
+    val redirectedCommand = s"$instantiatedCommandLine > $LocalStdoutValue 2> $LocalStderrValue"
```

Comment:
It was my understanding we wanted to capture stdout and stderr that "leaked" out of the command, not to intercept stdout and stderr that the command willfully redirected elsewhere.  Is that wrong?